### PR TITLE
FIX:  report render bem

### LINKED ISSUE #2611: BUG: report render bem broken if one surface is missing, a.k.a. homog
line 1752 returns a gray MRI image once one surface is not there.

cc @teonlamont @jaeilepp @agramfort

### PR DESCRIPTION
**Pull request by dengemann**
closes #2611

cc @agramfort @jaeilepp

### CODE REVIEW

**Comment by dengemann** [comment]
cryptic py35 error, if someone wants to look into that please go ahaead. Working with a local fix in the meantime.

**Comment by jaeilepp** [comment]
The same error appears in other PRs too

**Comment by Eric89GXL** [comment]
If we can't figure it out easily you could just add a skip for 3.5 for now with an XXX and we can get back to it. This really looks like a `mpl` / `3.5` issue to me.

File: mne/report.py [hunk 1]
```diff
@@ -1745,12 +1745,13 @@ def _render_bem(self, subject, subjects_dir, decim, n_jobs,
...
-                surf_fname = surf_fname[0]
+                surf_fnames.append(surf_fname[0])
             else:
                 warnings.warn('No surface found for %s.' % surf_name)
-                return self._render_image(mri_fname, cmap='gray')
-            surf_fnames.append(surf_fname)
-
+                continue
+        if len(surf_fnames) == 0:
+            warnings.warn('No surfaces found at all, rendering empyt MRI')
```
Comment (agramfort) [inline mne/report.py]: empyt-> empty

**Comment by agramfort** [comment]
besides LGTM

please fix rebase and merge

File: doc/whats_new.rst [hunk 2]
```diff
@@ -25,7 +25,8 @@ Changelog
 BUG
 ~~~
 
-    - None
+    Fix bug that prevented homo bem surfaces to be displayed in HTML reports by
```
Comment (agramfort) [inline doc/whats_new.rst]: keep the "-"

Comment (agramfort) [inline doc/whats_new.rst]: then squash and merge.

thanks

**Pull request merged**